src/uct/sm/mm/mm_sysv.c

```diff
@@ -46,8 +46,8 @@ uct_sysv_alloc(uct_md_h md, size_t *length_p, ucs_ternary_value_t hugetlb,
     }
 
     if (hugetlb != UCS_NO) {
-        status = ucs_sysv_alloc(length_p, address_p, flags | SHM_HUGETLB, &shmid
-                                UCS_MEMTRACK_VAL);
+        status = ucs_sysv_alloc(length_p, (*length_p) * 2, address_p,
```

Comment:
why *2?

Reply:
setting the maximal size an allocation can be expanded as 2x

Reply:
yes but why 2?